bicycleparameters/main.py

```diff
@@ -726,6 +728,271 @@ def com_symbol(ax, center, radius, color='b'):
+            If true inertia ellipses for each rigid body will be displayed.
+
+        Returns
+        =======
+        fig : matplotlib.pyplot.Figure
+
+        Notes
+        =====
+
+        If the flywheel is defined, it's center of mass corresponds to the
+        front wheel and is not depicted in the plot.
+
+        """
+        par = io.remove_uncertainties(self.parameters['Benchmark'])
+        parts = get_parts_in_parameters(par)
+
+        try:
+            slopes = io.remove_uncertainties(self.extras['slopes'])
+            intercepts = io.remove_uncertainties(self.extras['intercepts'])
+            penInertias = io.remove_uncertainties(
+                self.extras['pendulumInertias'])
+        except AttributeError:
+            pendulum = False
+
+
+        fig1 = go.Figure()
+
+        # define some colors for the parts
+        numColors = len(parts)
+        cmap = plt.get_cmap('gist_rainbow')
+        partColors = {}
+        
+     
+        
+        for i, part in enumerate(parts):
+            partColors[part] = cmap(1. * i / numColors)
+
+        if inertiaEllipse:
+            # plot the principal moments of inertia
+            for j, part in enumerate(parts):
+                I = inertia.part_inertia_tensor(par, part)
+                Ip, C = inertia.principal_axes(I)
+                if part == 'R':
+                    center = np.array([0., par['rR']])
+                elif part in 'FD':
+                    center = np.array([par['w'], par['rF']])
+                else:
+                    center = np.array([par['x' + part], -par['z' + part]])
+                # which row in C is the y vector
+                uy = np.array([0., 1., 0.])
+                for i, row in enumerate(C):
+                    if np.abs(np.sum(row - uy)) < 1E-10:
+                        yrow = i
+                # remove the row for the y vector
+                Ip2D = np.delete(Ip, yrow, 0)
+                        # remove the column and row associated with the y
+                C2D = np.delete(np.delete(C, yrow, 0), 1, 1)
+                        # make an ellipse
+                Imin = Ip2D[0]
+                Imax = Ip2D[1]
+                        # get width and height of a ellipse with the major axis equal
+                        # to one
+                unitWidth = 1. / 2. / np.sqrt(Imin) * np.sqrt(Imin)
+                unitHeight = 1. / 2. / np.sqrt(Imax) * np.sqrt(Imin)
+                        # now scaled the width and height relative to the maximum
+                        # principal moment of inertia
+                width = Imax * unitWidth
+                height = Imax * unitHeight
+                angle = -np.degrees(np.arccos(C2D[0, 0]))
+                x_center = center[0]
+                y_center = center[1]
+                x_ep, y_ep = plot.ell(x_center = x_center, y_center = y_center,ax1 = [np.cos(angle), np.sin(angle)], ax2 = [-np.sin(angle), np.cos(angle)],a = height, b = width)
```

Comment:
```suggestion
                x_ep, y_ep = plot.ell(x_center=x_center, 
                                      y_center = y_center,
                                      ax1=[np.cos(angle), np.sin(angle)],
                                      ax2 = [-np.sin(angle), np.cos(angle)],
                                      a = height, 
                                      b = width)
```

For long lines, do some wrapping so it looks better and stays within 79 character line lengths.